client/src/BookProfile/BookProfile.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from "react";
+import data from '../Mocks/books.json';
+import { Card } from 'antd';
+import './BookProfile.css';
+import Rater from 'react-rater';
+import { withRouter } from "react-router-dom";
+import defaultBook from '../pictures/bookSmile.png';
+
+const { Meta } = Card;
+
+export class BookProfile extends Component {
+
+    constructor(props) {
+        super(props);
+
+        let splitedPath = window.location.pathname.split("/");
+        let bookId = 1;
```

Comment:
why do we use hardcoded id, instead of using props?

Reply:
Its not hardcode it's default. If I won't set default then when somebody type incorrect  url e.g without id of book at the end there will be problem

Reply:
please rename it to defaultBookId then, if possible move it to const field.

Reply:
ok, leave bookId as it is state. Just create const with "1" inside.

Reply:
and then in Line 18 I will assign value which is not default to variable default ?? And to const ? hmmm

Reply:
no
`let bookId = defaultBookId;`

Generally if you have any magic number or string it should have a name